image-customize

```diff
@@ -52,91 +51,101 @@ def prepare_install_image(base_image, install_image, resize):
     return install_image
 
 
-def run_command(machine_instance, commandlist):
-    """Run command inside image"""
-    for foo in commandlist:
+class ActionBase(argparse.Action):
+    '''Keep an ordered list of actions'''
+
+    def __call__(self, parser, namespace, value, option_string=None):
+        if 'actions' not in namespace:
+            setattr(namespace, 'actions', [])
+        previous = namespace.actions
+        previous.append((self.execute, value))
+        setattr(namespace, 'actions', previous)
+
+
+class InstallAction(ActionBase):
+    """Install package into a test image
+    It could be done via local rpms or normal package installation
+    """
+    @classmethod
```

Comment:
Why `@classmethod`?  I could see an argument for making this a `@staticmethod`, but class method really makes no sense at all when every class has exactly one instance...

Reply:
Sure, but why not `@staticmethod` in that case, and drop the `cls` parameter?